### DISCUSSION TITLE
AndroidTV | اندروید تی‌وی

### DISCUSSION THREAD

**Discussion by kavehrahmati** [hiddify/hiddify-app#512] [Ideas]
Hi, is it possible to release androidTV compatible version of app, because it's not working fine now.Thanks
درود فراوان! آیا امکان انتشار نسخه سازگار با اندروید تی‌وی هم وجود دارد؟  با تشکر
#208

**Comment by moli1369** [comment]
#up

**Comment by Paliz64** [comment]
I agree

**Comment by Mammad3861** [comment]
Hi,
I tested Hiddify on some Android TV boxes and found that compatibility depends on the device CPU architecture.

For many older Iranian Android TV boxes, the `armeabi-v7a` (ARMv7) build may work better.
You can check your device architecture using apps like **CPU-Z** (or similar hardware information apps) and choose the matching APK version.

My tests:
- TSCO Android Box ✅
- Filimo Box ✅

Both worked correctly with the ARMv7 version.
Of course, an official Android TV optimized version with better remote control support would still be a great improvement.
Thanks for your work 🙏

---
سلام،
من Hiddify را روی چند اندروید باکس تست کردم و سازگاری آن به معماری CPU دستگاه بستگی دارد.
در بسیاری از اندروید باکس‌های قدیمی ایرانی، نسخه `armeabi-v7a` (ARMv7) ممکن است سازگاری بهتری داشته باشد.

برای بررسی معماری دستگاه می‌توان از برنامه‌هایی مثل **CPU-Z** یا ابزارهای مشابه اطلاعات سخت‌افزاری استفاده کرد و نسخه APK مناسب را انتخاب کرد.

تست‌های من:
- اندروید باکس تسکو ✅
- فیلیمو باکس ✅

هر دو با نسخه ARMv7 بدون مشکل کار کردند.
البته انتشار نسخه رسمی مخصوص Android TV با پشتیبانی بهتر از ریموت کنترل، تجربه کاربری را بهتر خواهد کرد.
ممنون از توسعه‌دهندگان 🙏
